debezium-server/debezium-server-distro/src/main/resources/assemblies/server-distribution.xml

```diff
@@ -17,9 +17,6 @@
           <useTransitiveFiltering>true</useTransitiveFiltering>
           <excludes>
             <exclude>org.apache.kafka:kafka-tools:*</exclude>
-            <exclude>com.github.luben:zstd-jni:*</exclude>
```

Comment:
Did you do due dilligence in terms of licenses?

Reply:
BSD/GPL

BTW they are also shipped by default in Kafka distribution

Reply:
Ok, so to clarify for future reference: the artifact in question is dual-licensed, also available under BSD which is fine.